From 93e7b8399e1dfe4d6645184efab1c70fa4268203 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Tue, 13 Apr 2021 19:06:59 +0200
Subject: [PATCH] Override settings only for Budgie

This left the default settings for GNOME, and override them only for the Budgie session. Also add override for the desktop background, because the default one is animated, which is not supported by Budgie background renderer.
---
 src/wm/20_solus-project.budgie.wm.gschema.override | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/wm/20_solus-project.budgie.wm.gschema.override b/src/wm/20_solus-project.budgie.wm.gschema.override
index f568a2c7..c2ad61cb 100644
--- a/src/wm/20_solus-project.budgie.wm.gschema.override
+++ b/src/wm/20_solus-project.budgie.wm.gschema.override
@@ -1,7 +1,10 @@
-[org.gnome.mutter]
+[org.gnome.mutter:Budgie]
 edge-tiling = true
 attach-modal-dialogs = true
 
-[org.gnome.desktop.wm.preferences]
+[org.gnome.desktop.background:Budgie]
+picture-uri='file:///usr/share/backgrounds/gnome/adwaita-day.png'
+
+[org.gnome.desktop.wm.preferences:Budgie]
 button-layout = 'appmenu:minimize,maximize,close'
 
-- 
2.31.1

